integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -269,13 +269,17 @@ def get_grades_summary(
         course_details,
         enterprise_enrollment,
         channel_name,
+        incomplete_count=None,
```

Comment:
Remind me who the consumer of this is and why it's getting added?

Reply:
it's used to decide when to fill in the date automatically, among other things, later in the method. for audit we only want to fill if course is complete based on `incomplete_count==0`

Reply:
we don't use it for non audit right now, I think we can further refactor so audit based code is compeletely separate from non audit...